panel_client/run_panel.py

```diff
@@ -24,6 +24,10 @@ def main():
       '--panel_type',
       default='KeyboardPanel',
       choices=[panel.__name__ for panel in PANELS])
+  parser.add_argument(
+      '--player_number',
```

Comment:
Need to pass the parsed value through to the keyboard panel now :)

Reply:
Done. But P2's controls have an error in them somewhere. PTAL. 

It'd be helpful if the server would either print or send back parsing errors.

Reply:
What do you mean have an error in them? The game works for me at least as of https://github.com/wearhere/spacecontrol/pull/17.

Yeah validation would be nice. Filed an issue [here](https://github.com/wearhere/spacecontrol/issues/18), feel free to add rules!